Flow.Launcher/MainWindow.xaml.cs

```diff
@@ -84,8 +84,28 @@ private void OnLoaded(object sender, RoutedEventArgs _)
                             QueryTextBox.SelectAll();
                             _viewModel.LastQuerySelected = true;
                         }
+
                     }
+                    else
+                    {
+                    }
+                }
+                else if (e.PropertyName == nameof(MainViewModel.ProgressBarVisibility))
+                {
+                    Dispatcher.Invoke(() =>
+                    {
+                        if (ProgressBar.Visibility == Visibility.Hidden)
```

Comment:
shouldnt this be MainViewModel.MainWindowVisibility? i thought we are resuming and pausing progress bar animation based on the visibility of mainwindow

Reply:
> what happens when the query is still loading though? like if its a long running query, i click on some other window and come back a minute later, this change means the bar will stop if it's hidden despite the query is still running right?

That's the reason changing this to progressbar visibility